charts/fpl-case-service/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: fpl-case-service
 apiVersion: v1
 home: https://github.com/hmcts/fpl-ccd-configuration
-version: 1.6.0
+version: 1.7.0
```

Comment:
Remember to create PR for https://github.com/hmcts/cnp-flux-config repo as well

Reply:
sure will do it once it is merged to master